### PR TITLE
Minor ETS changes

### PR DESCRIPTION
**Pull request by doomspork**
Going through the ETS lesson again and wanted to reword a few things.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Elixir School [![License](http://img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
 
-> Lessons in the Fundamentals of Elixir, inspired by Twitter's [Scala School](http://twitter.github.io/scala_school/).
+> Lessons in Elixir programming, inspired by Twitter's [Scala School](http://twitter.github.io/scala_school/).
```
Comment (milmazz) [inline README.md]: Lessons about the Elixir programming language, inspired by...

File: README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ _Feedback and participation is welcome. Please see [Contributing](CONTRIBUTIING.
 
   ```md
   title: Elixir School
-  description: Lessons in the Fundamentals of Elixir
+  description: Lessons in Elixir programming
```
Comment (milmazz) [inline README.md]: Lessons about the Elixir programming language

File: README.md (hunk 1 above)
Comment (doomspork) [inline README.md]: Much better, thanks @milmazz  :+1:

I'll make that change throughout.

File: index.md [hunk 3]
```diff
@@ -6,7 +6,13 @@ lang: en
 
 [![License](http://img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
 
-Lessons in the Fundamentals of Elixir, inspired by Twitter's [Scala School](http://twitter.github.io/scala_school/).
+Lessons about the Elixir programming language, inspired by Twitter's [Scala School](http://twitter.github.io/scala_school/).
+
+Available in [Chinese][cz], [Spanish][es], and [Japanese][jp].
```
Comment (doomspork) [inline index.md]: Should these be translated into their native languages?  For example (using Google Translate):

Available in [汉语][cz], [Español][es], and [日本語][jp].

[cz]: https://elixirschool.com/cz/
[es]: https://elixirschool.com/es/
[jp]: https://elixirschool.com/jp/

Comment (milmazz) [inline index.md]: :+1:

**Pull request merged**